configuration/cdap-site.xml

```diff
@@ -645,6 +674,37 @@
   </property>
 
   <property>
+    <name>data.tx.prune.enable</name>
+    <value>true</value>
+    <description>Enable invalid transaction list pruning</description>
+    <display-name>Transaction Client Prune Enable</display-name>
+    <value-attributes>
+      <type>boolean</type>
+    </value-attributes>
+ </property>
+
+ <property>
+    <name>data.tx.prune.plugins</name>
+    <value>data.tx.pruning.plugin</value>
+    <description>List of transaction pruning plugins; for CDAP HBase tables that use transaction functionality to skip or clean invalid data</description>
+    <display-name>Transaction Client Prune Plugins</display-name>
+ </property>
```

Comment:
Remove this property, per [here](https://issues.cask.co/browse/CDAP-8393?focusedCommentId=28133&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-28133)